tutor/src/helpers/analytics.js

```diff
@@ -64,12 +64,30 @@ const Translators = {
   },
 };
 
+const TAG_MANAGER_ID = 'GTM-W6N7PB';
```

Comment:
is it ok to expose this tag? do we have any environment variables?

Reply:
yah it's ok to expose it since it's included in every post to google.  I thought about making a config for it but doubt it'll ever change so decided to just do this